Gemfile.lock

```diff
@@ -266,4 +267,4 @@ RUBY VERSION
    ruby 2.6.6p146
 
 BUNDLED WITH
-   2.2.11
```

Comment:
Was the "BUNDLED WITH" version changed automatically? Just curious if I need to change mine to 2.2.16 as well.

Reply:
I have no idea about that version, I think the bundle install version on yours might be different to mine. If you tried update your bundle install it might get the latest version.  I don’t think you should touch the gem.lock file. May be a TA could help us